packages/client/test/open-feature.spec.ts

```diff
@@ -51,12 +81,22 @@ describe('OpenFeature', () => {
         .clearHooks()
         .setLogger(console)
         .getClient();
-        
-  
+
+
       expect(client).toBeDefined();
     });
 
-    
+
   });
 
+  describe('Requirement 1.6.1', () => {
+    it('MUST define a `shutdown` function, which, when called, must call the respective `shutdown` function on the active provider', () => {
+      OpenFeature.setProvider(MOCK_PROVIDER);
+      expect(OpenFeature.providerMetadata.name).toBe('mock-events-success');
+      OpenFeature.close().then(
+        () => {
+          expect(MOCK_PROVIDER.onClose).toHaveBeenCalled();
```

Comment:
Since this test isn't async, and also there's no `done()` being used, this test will end before the expectation runs.

You can either make it async and await the promise, or put a called to `done()` inside the `then`.